.licenserc.yaml

```diff
@@ -20,5 +20,7 @@ header:
     - 'OMICRON_VERSION'
     - '.husky'
     - 'mockServiceWorker.js'
+    - 'app/util/format-date-range.ts'
+    - 'app/util/format-date-range.test.ts'
```

Comment:
This lib is MIT licensed. Unsure if there's a better way to handle this – it's so small it feels better to copy it over rather than importing.